advanced-integration/v2/backend/php/CONTRIBUTING.md

```diff
@@ -0,0 +1,14 @@
+# How to Contribute
```

Comment:
It doesn't look like this example uses Slim, was this pulled over from a previous/different example?  We can probably just remove the whole thing.